website/versioned_docs/version-21.1.1/api/puppeteer.elementhandle.aselement.md

```diff
@@ -1,17 +0,0 @@
----
```

Comment:
Can you add a brief comment explaining the purpose of this method?

Reply:
In the past, this method had a purpose. Now you can do an `instanceof` check, so it's obsolete.